app/elements/pw-shell.html

```diff
@@ -223,6 +251,14 @@
       _searchParams: {
         type: Object,
         observer: '_searchParamsChanged'
+      },
+
+      /**
+       * Which version (1.0 or 2.0) of the docs we are using, so that the
+       * main nav and side nav can be kept in sync across page navigations.
+       */
+      activeDocsVersion: {
+        type: String
```

Comment:
Also: as chatted offline: not all URLs have a version, so that's not always true